### DISCUSSION TITLE
Need help with responsive flex form

### DISCUSSION THREAD

**Discussion by romsar** [tailwindlabs/tailwindcss#5191] [Help]
https://jsfiddle.net/rom4i/3fvzx51g/21/

I have flexbox form, that have two price fields that can grow and three more fields without flex-grow.
If user screen width is small it climbs over the right side. How to make it wrap? If delete flex-nowrap and set flex-wrap price fields will have fixed width.

**Comment by Martleby** [comment]
On the nested flex elements (like the Price 1 and Price 2 sections) you need add some breakpoints like:

`class="block md:flex"` 

To stop them flexing at smaller screen widths. Also in these situations I wouldn't rely on flex wrap on the main container and instead either flex-col or just block depending on the screen size.

Yours is quite a complicated example, but here is a small play that demonstrates what I mean: https://play.tailwindcss.com/ZGLJQowuPz Note how Price 2 behaves.

Cheers,
Martin.
